app/src/main/res/values/arrays.xml

```diff
@@ -11,6 +11,7 @@
         <item>HOTP</item>
         <item>Steam</item>
         <item>Yandex</item>
+        <item>MOTP</item>
```

Comment:
What is the correct way to abbreviate Mobile-OTP? Is it mOTP or MOTP?

Reply:
I think it should be MOTP, however I've seen mOTP quite a lot so there doesn't seem to be great consensus :(

Reply:
Alright, let's leave it at "MOTP" and wait to see if someone complains about it.